tests/test_fio_workload.py

```diff
@@ -16,12 +17,38 @@
 logger = logging.getLogger(__name__)
 
 
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    Fixture only for teardown
+    """
+
+    self = request.node.cls
+
+    def finalizer():
+        teardown(self)
+    request.addfinalizer(finalizer)
+
+
+def teardown(self):
+    logger.info(f"Deleting pod {self.pod_obj.name}")
+    self.pod_obj.delete()
+    self.pvc_obj.reload()
+    pv = pvc.get_pv_for_pvc(self.pvc_obj)
+    logger.info(f"PV for PVC is {pv.name}")
+    logger.info(f"Deleting PVC {self.pvc_obj.name}")
+    self.pvc_obj.delete()
+    logger.info(f"Deleting PV {pv.name}")
+    pv.delete()
+
+
 @pytest.mark.usefixtures(
     create_rbd_secret.__name__,
     create_ceph_block_pool.__name__,
     create_rbd_storageclass.__name__,
     create_pvc.__name__,
-    create_pod.__name__
+    create_pod.__name__,
+    test_fixture.__name__,
```

Comment:
it's a bit confusing to have fixtures that do one operation each and a fixture that does multiple operations (the teardown)

Reply:
Done, seperated teardown for PVC and POD